plugins/faustwp/includes/graphql/callbacks.php

```diff
@@ -44,6 +45,28 @@ function register_templates_field() {
 	);
 }
 
+add_filter( 'graphql_get_setting_section_field_value', __NAMESPACE__ . '\\filter_introspection', 10, 5 );
+
+function filter_introspection( $value, $default_value, $option_name, $section_fields, $section_name ) {
```

Comment:
We should probably document this function.

Tip: When leveraging a WordPress filter, you can typically search the codebase where the filter is being implemented and copy those filter parameters.

In this case, I did a search [for the filter string](https://github.com/search?q=repo%3Awp-graphql%2Fwp-graphql%20graphql_get_setting_section_field_value&type=code) and [located it here](https://github.com/wp-graphql/wp-graphql/blob/develop/access-functions.php#L837C1-L845C5).

```suggestion
add_filter( 'graphql_get_setting_section_field_value', __NAMESPACE__ . '\\filter_introspection', 10, 5 );
/**
 * Update WPGraphQL's introspection setting's label in order inform a user of the exception for Faust.
 *
 * @param mixed  $value          The value of the field
 * @param mixed  $default_value  The default value if there is no value set
 * @param string $option_name    The name of the option
 * @param array  $section_fields The setting values within the section
 * @param string $section_name   The name of the section the setting belongs to
 */
function filter_introspection( $value, $default_value, $option_name, $section_fields, $section_name ) {
```

Reply:
Done in https://github.com/wpengine/faustjs/pull/1594/commits/9e13dbb6040c0c4ff1241e5fed3516fde91022eb